src/core/server/config/config_service.test.ts

```diff
@@ -28,11 +28,12 @@ jest.mock('../../../legacy/utils/package_json', () => ({ pkg: mockPackage }));
 import { schema, Type, TypeOf } from '@kbn/config-schema';
 
 import { ConfigService, Env, ObjectToConfigAdapter } from '.';
-import { logger } from '../logging/__mocks__';
+import { loggingServiceMock } from '../logging/logging_service.mock';
 import { getEnvOptions } from './__mocks__/env';
 
 const emptyArgv = getEnvOptions();
 const defaultEnv = new Env('/kibana', emptyArgv);
+const logger = loggingServiceMock.create();
```

Comment:
I decided to keep the previous name and unify this approach in files. Any suggestions about naming?

Reply:
I'm not very good at naming things, but this seems reasonable to me. Consistency is really the thing I care about, personally.